### DISCUSSION TITLE
Does anyone use ProRes RAW on the regular and have a good FCP workflow for that media?

### DISCUSSION THREAD

**Discussion by Gustafa7** [CommandPost/FCPCafe#204] [Final Cut Pro Questions] [answered]
I can shoot ProRes RAW codec all day long and I jumped on the bandwagon early for the wish of staying inside FCP as much as possible. Roundtripping is fun and everything, but more and more I'm finding myself really liking what Resolve can do with CST/Color Space Transform to bring the depth of pixels out, treat gamma with some respect etc.

FCP seems so limited to just slapping a log to 709 LUT and go about some slight wheel work and thats it. I dont want to stack LUT on LUT etc, OR maybe its just my ignorance and Im missing a better/proper way to handle my footage?

Thanks for your input and help, let alone kindess, Cafe Community.

**Comment by latenitefilms** [comment]
Good question! I'll try get some colourists to answer this, as I don't want to give any incorrect/misguided advice.

**Reply by Gustafa7** [reply]
Hey, thanks man! I really love the codec, but it lives in a very tight ecosystem and converting to CDNG just defeats the point.

**Reply by latenitefilms** [reply]
Someone replied with this: https://twitter.com/sightcreations/status/1671277838933082112

Also, you might want to look into Assimilate Play Pro Studio if you want to go from ProRes RAW to DaVinci Resolve.

**Reply by serralha** [reply]
On last work that have ProRes RAW I use RAW Convertor https://apps.apple.com/pt/app/raw-convertor/id1598580439?l=en&mt=12
ProRes Raw to Cinema DNG, edit on FCP and graded on DaVinci Resolve. Can't understand why both Blackmagic and Apple still have this RAW "restrictions"... At least, now we have BRAW Toolbox just missing PRORES Toolbox :)

**Reply by Gustafa7** [reply]
Hey serralha, yes, good ole RAW converter, its been in my toolbox for years. I was really hoping to avoid cdng and just stay in FCP, but I have done this as well. It has its merits, Im just tired of having so many versions of the files just for small shoots, etc.

**Comment by StuHoop** [comment]
Hi [Gustafa7], we shoot mostly prores raw and do most of our finishing inside Final Cut.  We supplement the built in tools with Color Finale and with Dehancer.  For quick things, letting Final Cut do the Raw to Log Conversion to Vlog, but not applying their vlog camera lut, then using Color Finale's ACES workflow does the trick.  

Otherwise I'll turn both the Raw to Log Conversion and Camera Lut off and just work with the linear raw.  I usually still use Color Finale's sliders and curves before Final Cut's as they go a little deeper, but for some masking and stuff I'll use final cut's color wheels as well.  Dehancer is more of a stylistic finish but I find it works very well on the raw footage.  

I'm sure Resolve out performs FCPX + Color Finale but together they give me a toolset that does what I need while letting me keep the speed of the Apple and ProResRaw workflow.

**Reply by Gustafa7** [reply]
StuHoop, 

I used to run ColorFinale almost exclusively but when my ColorFinale Pro license expired a few years back, I never renewed. I really the Vectors when that was introduced, blah blah blah. Maybe I should dig back into that. They sold to another company a while ago, didnt they? 

Thanks for giving your input, Im trying to keep it all in FCP if so, unless I get a budget for color work that I can send out, thats a different story.

**Reply by StuHoop** [reply]
There was a time when Apple caught up and ColorFinale was redundant, then I think 2 came out and trackable masks and more features and color management made it a must use again for me.  Now Apple has mask tracking and such too, but I still use Color Finale for a lot of weird shaped masks and other scenarios.  Anything shot in Vlog I use it for ACES, anything shot Raw I still use the sharpening and color wheels from Color finale.

**Comment by MazzeAderhold** [comment, marked as answer]
It depends on your workflow.
While a lot of people fancy going ProRes RAW ==> cDNG, in order to keep the data 'RAW', that workflow also has its caveats.
Starting with Resolve not offering a great deal of debayering options in terms of color space and EOTF.
Plenty of people struggle getting the color right, or the shot in general to respond nicely to the color wheels.

The alternative is to transcode ProRes RAW to conventional ProRes 4444 XQ.
AFor this task (as well as transcoding to cDNG) you can use Assimilate Play Pro Studio on macOS and WIndows.
(https://www.assimilateinc.com/products/playprostudio/ ) - 
there is a coupon code "PPS50OFF" that will give you 50% off the retail price for a permanent license.

The ProRes workflow has the advantage of being able to render out high quality online files in the native camera space (i.e. V-Gamut / V-Log or S-Gamut 3.cine / S-Log3), that you can then natively work on in Resolve.

I like to remind people that ARRI's Alexa was not the big success it was because of ARRI RAW, but because it offered an easy and fast ProRes-based workflow. There is tons of image data in those ProRes files - and there's tons of image information - probably much more than you require anyways - in the ProRes files you can create from the ProRes RAW files. 
Although the filesizes might be a little bit bigger than those of the ProRes RAW originals, the workflow is straight forward, better defined and the picture is easier to grade and manage. Plus, a single ProRes file is easier on storage, and copy processes than an image sequence, which DNG is.

For those who really need WB adjustment - they can perfectly take care of this inside Play Pro Studio, before rendering out ProRes masters.
Because outside of that, I barely see the neccessity to stay RAW, even on final grading.
Using ProRes as the acquisition and mastering file format has worked for 95% of all shows that were shot on ARRI Alexa.
It should thus also work for the majority of content that is being shot using an Atomos recorder ;-) .

My 0.02$.


Cheers,
Mazze

**Reply by Gustafa7** [reply]
Mazze, theres absolutely nothing to argue here, I 100% agree with everything you said. I may have just reached the limitations of the PR RAW and I know it was mostly Apples response to licensing, etc. Im glad I have it as an option, but yea, PR 4444 XQ is straight fire and I love it, just was hoping to dig further into another codec and I have some clients that allow me to do so. Thanks for the push in the right direction, I truly appreciate your input!

Cheers,
Chase

**Reply by joema4** [reply]
We have tested ProRes RAW a lot from our Sony FX6 cameras and Atomos, and I don't see the benefit over regular ProRes 422 or the internal XAVC-I codec. Yes the specs are different but I don't see a real-world difference.

This is especially true now that Resolve 18.5 has RAW controls for Sony XAVC-I. You can make virtually the same adjustments to WB, ISO, color temp, etc. as you can with ProRes RAW. I did simultaneous internal/external recording on a test chart and corrected WB from across a huge range and I can barely see any difference. 

Below is a link to full-res TIFF frame exports from FCP and Resolve Studio 18.5 beta. The FX6 ProRes RAW files were adjusted for WB in FCP; the XAVC-I files were adjusted for WB using Resolve's new RAW controls for XAVC-I. All I did in Photoshop was annotate them, keeping them as TIFF. Unfortunately, Resolve 18.5 beta only has RAW controls for Sony XAVC-I. 

https://www.dropbox.com/sh/1872lyvuth4e0g0/AAAjly80wdNWjgZOPzRDnI_Ia?dl=0

The conventional reasoning is true RAW gives access to the RGGB mosaic, which is hypothetically before WB and chroma sub-sampling exist. By the time the image reaches an encoded RGB format, WB is baked in. However if the NLE vendor is given enough data about the specific camera's color science and image pipeline, and if the metadata is rich enough, in theory it can be manipulated almost like RAW.
